lib/media/drm_engine.js

```diff
@@ -1113,13 +1113,21 @@ shaka.media.DrmEngine = class {
     const keys = [];
     const keyIds = [];
 
-    clearKeys.forEach((keyHex, keyIdHex) => {
-      const keyId = Uint8ArrayUtils.fromHex(keyIdHex);
-      const key = Uint8ArrayUtils.fromHex(keyHex);
+    clearKeys.forEach((key, keyId) => {
```

Comment:
I think you could just rename key, keyId to k, kid if you want.  It's not that important, but the renaming in local variables in the callback seems weird to me.